boot_partition.txt

```diff
@@ -0,0 +1,812 @@
+[   57467608]  ./flatcar/vmlinuz-a*
+[     949440]  ./EFI/boot/bootx64.efi*
+[     855912]  ./EFI/boot/mmx64.efi*
+[     783864]  ./flatcar/grub/x86_64-efi/core.efi.signed*
+[     783864]  ./EFI/boot/grubx64.efi*
+[     782336]  ./flatcar/grub/x86_64-efi/core.efi*
```

Comment:
these are all the same file and i want to say that we only need `EFI/boot/grubx64.efi`, but this needs testing.

Reply:
Unless we would also remove them in the update postinst hook these would still occupy space to prevent an update - but even then for some old clients the postinst hook might be too late because the kernel can't even be extracted to `/boot`.